eox_core/edxapp_wrapper/backends/enrollment_h_v1.py

```diff
@@ -46,11 +55,15 @@ def check_edxapp_enrollment_is_valid(*args, **kwargs):
     course_id = kwargs.get("course_id")
     force_registration = kwargs.get('force_registration', False)
     mode = kwargs.get("mode")
+    if not kwargs.get("email") and not kwargs.get("username"):
+        return ['Email or username needed']
+    if kwargs.get("email") and kwargs.get("username"):
+        return ['Email or username but not both']
```

Comment:
A bit more context in the message would be better: 'You have to provide an email or username but not both'

Reply:
Ok done